pandas/tests/frame/test_alter_axes.py

```diff
@@ -171,21 +173,6 @@ def test_assign_columns(self, float_frame):
         tm.assert_series_equal(float_frame["C"], df["baz"], check_names=False)
         tm.assert_series_equal(float_frame["hi"], df["foo2"], check_names=False)
 
-    def test_set_index_preserve_categorical_dtype(self):
```

Comment:
I moved this to sit with the only other categorical test (the one I'm adding) in `TestCategoricalIndex` class